lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -89,7 +89,10 @@ def execute(powershell_code, timeout_ms = nil, working_dir = nil)
             (prop.text.nil? ? nil : Base64.decode64(prop.text))
           # if err contains data it must be "real" stderr output
           # which should be appended to what PS has already captured
-          value += err if err && (err != []) && (name == 'stderr')
+          if name == 'stderr'
+            value = value.nil? ? [] : [value]
+            value += err if !err.nil?
```

Comment:
I made notes in commit message... our previous behavior was to pass an array of strings up through "stderr" - but I accidentally broke it in fa3d618.  There were no tests in the provider to catch it.

We should probably rethink that at some point (to just use a string value everywhere, like we do for `stdout`) - but I'm trying to preserve what we were doing.